sdk/src/main/java/io/customer/sdk/di/CustomerIOComponent.kt

```diff
@@ -71,7 +66,7 @@ class CustomerIOComponent(
         )
 
     val logger: Logger
-        get() = override() ?: LogcatLogger(sdkConfig)
+        get() = sharedComponent.logger
```

Comment:
a little confused here, should we keep the ability to override the logger? would there be any benifit? for example in test?

Reply:
That's a good catch 😅 

Although, the only benefit I can think of doing this is; any tests using shared dependencies would be unchanged and will not having to override in shared instance separately, as doing it in current instance will work for them. Which makes sense to me. As long as we are using these dependencies from `CustomerIOComponent` instead of `CustomerIOSharedComponent` directly. I'll update this soon.